zerver/lib/bugdown/__init__.py

```diff
@@ -1421,23 +1421,36 @@ def handleMatch(self, match: Match[str]) -> ElementStringNone:
         db_data = self.markdown.zulip_db_data
         return url_to_a(db_data, url)
 
-class UListProcessor(markdown.blockprocessors.UListProcessor):
-    """ Process unordered list blocks.
-
-        Based on markdown.blockprocessors.UListProcessor, but does not accept
-        '+' or '-' as a bullet character."""
+class SaneOListProcessor(markdown.blockprocessors.OListProcessor):
+    """ Process ordered list blocks.
 
-    TAG = 'ul'
-    RE = re.compile('^[ ]{0,3}[*][ ]+(.*)')
+        Based on the sane_list extension with tab_length set to 2.
```

Comment:
Can you explain why we're not inheriting from the upstream `sane_list` extension directly here?

Reply:
The extension is only a few lines so I thought we might as well just inherit from the base O/UListProcessor. Its totally fine to inherit from sane_list too.

Reply:
Yeah, let's do that -- I think it makes our intent more readable.

Reply:
Ok did that.